dvc/repo/data.py

```diff
@@ -157,7 +157,9 @@ def _git_info(scm: "Base", untracked_files: str = "all") -> GitInfo:
     return GitInfo(
         staged=staged,
         unstaged=unstaged,
-        untracked=untracked,
+        untracked=untracked
+        if os.name != "nt"
+        else [posixpath_to_os_path(path) for path in untracked],
```

Comment:
This hurts to read 😄 Just use a temp var, please.

Reply:
Agreed :smile: